src/esp/assets/ResourceManager.cpp

```diff
@@ -1610,8 +1639,13 @@ void ResourceManager::addPrimitiveToDrawables(int primitiveID,
                                               scene::SceneNode& node,
                                               DrawableGroup* drawables) {
   CHECK(primitive_meshes_.count(primitiveID));
-  createGenericDrawable(*primitive_meshes_.at(primitiveID), node, NO_LIGHT_KEY,
-                        WHITE_MATERIAL_KEY, drawables);
+  gfx::Drawable::Flags meshAttributeFlags{};
```

Comment:
Wait, why do you empty the mesh attributes flags here? Shouldn't you repopulate them using the semantic mesh?

Reply:
This is inside the function `addPrimitiveToDrawables`, which has nothing to do with the semantic mesh.
I hope I did not misunderstand anything here.